libraries/Microsoft.Bot.Streaming/Payloads/Disassemblers/PayloadDisassembler.cs

```diff
@@ -120,12 +122,12 @@ private Task SendAsync()
                 isLengthKnown = true;
             }
 
-            Sender.SendPayload(header, Stream, isLengthKnown, OnSent);
+            Sender.SendPayload(header, Stream, isLengthKnown, OnSentAsync);
 
             return _taskCompletionSource.Task;
         }
 
-        private async Task OnSent(Header header)
+        private async Task OnSentAsync(Header header)
```

Comment:
>OnSentAsync [](start = 27, length = 11)

cancelationToken

Reply:
Can't add the paramter without breaking the interface, we will need to refactor this as a separate effort if we want to support cancellationTokens.